bundle/edu.gemini.wdba.xmlrpc.server/src/test/java/edu/gemini/wdba/tcc/InstrumentSupportTestBase.java

```diff
@@ -53,6 +62,31 @@ public void store() throws Exception {
         }
     }
 
+    protected TargetEnvironment getTargetEnvironment() {
+        final ISPObsComponent oc = SPTreeUtil.findTargetEnvNode(obs);
+        @SuppressWarnings("ConstantConditions")
+        final TargetObsComp toc  = (TargetObsComp) oc.getDataObject();
+        return toc.getTargetEnvironment();
+    }
+
+    protected void setTargetEnvironment(TargetEnvironment env) {
+        final ISPObsComponent oc = SPTreeUtil.findTargetEnvNode(obs);
+        @SuppressWarnings("ConstantConditions")
+        final TargetObsComp toc  = (TargetObsComp) oc.getDataObject();
+        toc.setTargetEnvironment(env);
+        oc.setDataObject(toc);
+    }
+
+    protected void addGuideStar(GuideProbe probe) {
+        TargetEnvironment env = getTargetEnvironment();
+        GuideGroup        grp = env.getPrimaryGuideGroup();
+        if (grp.isAutomatic()) {
+            grp = GuideGroup.create("Manual Group");
```

Comment:
Just  a tiny detail, on upstream these were renamed to "Manual" using  `GuideGroup.ManualGroupDefaultName`